src/vdom.rs

```diff
@@ -303,12 +303,14 @@ impl VdomInnerExclusive {
                     listeners: new_listeners,
                     attributes: new_attributes,
                     children: new_children,
+                    namespace: _,
                 }),
                 Node::Element(ElementNode {
                     tag_name: old_tag_name,
                     listeners: old_listeners,
                     attributes: old_attributes,
                     children: old_children,
+                    namespace: _,
```

Comment:
Shouldn't this check that both elements are in the same namespace, and if not, replace the old element with the new one?